docs/datasets-panel.rst

```diff
@@ -0,0 +1,104 @@
+==============
+Datasets panel
+==============
+Dataset panel is the central place to manage your datasets. Datasets are available to all sessions of a user. A user could choose to share dataset with other users.
+
+
+Upload
+======
+.. image:: img/datasets_panel-upload_button.png
+
+This is the initial state of dataset panel.
+You could use this upload button to upload dataset. See :doc:`/getting-started` for more detail.
+
+|
+|
+
+Dataset actions
+===============
+
+Visualize dataset
+-----------------
+.. image:: img/datasets_panel-globe_button.png
+
+Click the globe button after the dataset name to add the dataset to layer panel and visualize the dataset.
```

Comment:
I think we don't have to say click on the button. We can rather use something like "the globe button adds the dataset to the layer panel and plots the dataset."